tests/test_cluster.py

```diff
@@ -2285,5 +2285,22 @@ def test_update(self):
         with self.assertRaises(SystemExit):
             qds.main()
 
+class TestClusterACL(QdsCliTestCase):
+    def test_get_object_policy(self):
+        sys.argv = ['qds.py', 'cluster', 'object_policy', 'get', '--cluster-id', '1001']
+        print_command()
+        Connection._api_call = Mock(return_value={})
+        qds.main()
+        Connection._api_call.assert_called_with('GET', 'object_policy/get_object_policy', params ={'source_id':'1001',
```

Comment:
`get/update/delete_object_policy` methods are for UI, as UI didnt wanted to do 2 calls for a proper CRUD. We do've proper CRUD apis available for Object_policy as well. So idle calls chain would be like `GET cluster/:cid/object_policy` and then `GET object_policy/:opid`.. but as we didn't added those `get object_policy` methods to all the resources, what @harshshah87 did is fine.